README.md

```diff
@@ -129,7 +129,15 @@ points declared in `setup.py` — otherwise they will not be loaded by Plone:
 ```
 
 Also make sure that your add-ons do not use `zc3.autoinclude` in their `configure.zcml`
-configuration file.
+configuration file.  Instead, list the packages to include in your add-on's
+configure.zcml` explicitly:
+
+```
```

Comment:
We can get syntax highlighting, but only if the snippet is valid for the named language. That means removing both of the `...`s from the snippet.

````suggestion
```xml
````